plugins/soundsourcem4a/soundsourcem4a.cpp

```diff
@@ -126,6 +110,12 @@ Result SoundSourceM4A::tryOpen(const AudioSourceConfig& audioSrcCfg) {
         return ERR;
     }
 
+    m_framesPerSampleBlock = MP4GetTrackFixedSampleDuration(m_hFile, m_trackId);
+    if (MP4_INVALID_DURATION == m_framesPerSampleBlock) {
+      qWarning() << "No support for variable duration track samples";
```

Comment:
The warning does somehow not match the if condition. 
Can you add a comment to the code. Why this warning is issued here?